### PR TITLE
Small notes on $http_req(timeout) usage.

### PR DESCRIPTION
**Pull request by samael33**
#### Pre-Submission Checklist

- [x] Commit message has the format required by CONTRIBUTING guide
- [x] Commits are split per component (core, individual modules, libs, utils, ...)
- [x] Each component has a single commit (if not, squash them into one commit)
- [x] No commits to README files for modules (changes must be done to docbook files
in `doc/` subfolder, the README file is autogenerated)

#### Type Of Change
- [x ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:

- [ ] PR should be backported to stable branches
- [x] Tested changes locally
- [ ] Related to issue #XXXX (replace XXXX with an open issue number)

#### Description

Just small notes on $http_req(timeout) usage.

### CODE REVIEW

File: src/modules/http_async_client/doc/http_async_client_admin.xml [hunk 1]
```diff
@@ -435,7 +435,7 @@ xlog("L_INFO", "received reply for query $http_req_id\n");
 		<listitem><para><emphasis>hdr</emphasis>: sets/modifies/removes a HTTP header. N.B.: setting this variable multiple times will add several headers to the query.</para></listitem>
 		<listitem><para><emphasis>body</emphasis>: sets/modifies/removes the request body</para></listitem>
 		<listitem><para><emphasis>method</emphasis>: sets the HTTP method: either "GET", "POST", "PUT" or "DELETE" (these are the supported methods). (Note: if the method is not set, curl will use GET, or POST if a body is specified)</para></listitem>
-		<listitem><para><emphasis>timeout</emphasis>: sets the HTTP timeout</para></listitem>
+		<listitem><para><emphasis>timeout</emphasis>: sets the HTTP timeout. (Note, this timeout should be normally less, than <literal>tm.fr_timer</literal> timeout, cause transaction timeout has a higher priority over HTTP timeout)</para></listitem>
```
Comment (grumvalski) [inline src/modules/http_async_client/doc/http_async_client_admin.xml]: s/cause/because/
and 
s/less,/less/

Comment (samael33) [reply src/modules/http_async_client/doc/http_async_client_admin.xml]: Thanks, fixed

**Comment by miconda** [comment]
@samael33, @grumvalski -- the commit messages on this pull are really inappropriate for development history.

Next time check the contribution guidelines:

  * https://github.com/kamailio/kamailio/blob/master/.github/CONTRIBUTING.md

**Comment by grumvalski** [comment]
You're right, I missed it sorry.

Federico

**Comment by samael33** [comment]
I'm sorry as well, just was too excited on my first commit.

**Comment by miconda** [comment]
I rebased, edited the commits and then did a forced push -- worked this time, but this is usually dangerous to do, can break the repo. Better have the commit messages written properly.

**Comment by samael33** [comment]
Thanks!

Regards, Igor

**Pull request merged**